common/source/docs/common-rpm-based-notch.rst

```diff
@@ -0,0 +1,14 @@
+.. _common-rpm-based-notch:
+
+====================================
+RPM Sensor Based Dynamic Notch Setup
+====================================
+
+- Set :ref:`INS_HNTCH_ENABLE <INS_HNTCH_ENABLE>` = 1 to enable the harmonic notch
+- Set :ref:`INS_HNTCH_REF <INS_HNTCH_REF>` = 1 to set the harmonic notch reference value to unscaled, unless the RPM sensor is reporting a fraction of the actual rotor speed. This might occur in a geared drive train setup. If the reported RPM is 1/3 the actual, for example, set this to 0.33.
+- Set :ref:`INS_HNTCH_FREQ <INS_HNTCH_FREQ>` = below the hover frequency - you can easily determine this by performing a gentle hover and looking at the RPM sensor log data (RPM.rpm1)
+- Set :ref:`INS_HNTCH_BW <INS_HNTCH_BW>` = half of INS_HNTCH_FREQ
+
+Checking Harmonic Notch Effectiveness
+=====================================
```

Comment:
This is a general section that belongs elsewhere

Reply:
both esc and rpm (and eventually in-flight FFT) should  (esc doesnt...will add) have this section saying that post flight harmonic notch effectiveness checking is not required, but...if you want, use the method in the throttle based section to do so....this needs a link added to : ref:`common-imu-notch-filtering-post-configuration-flight-and-post-flight-analysis` just like the heli section and esc section have....so that if someone follows those paths they get the link to the method of checking effectiveness and will add the section to esc based also...

Reply:
updated